<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_Lyrics</class>
 <widget class="QWidget" name="GUI_Lyrics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>521</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" rowspan="2" colspan="2">
    <layout class="QGridLayout" name="gridLayout_11">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="6">
      <widget class="QLabel" name="lab_server">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Server</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_title"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lab_artist">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Artist</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lab_tit">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_artist"/>
     </item>
     <item row="0" column="7">
      <widget class="QComboBox" name="combo_servers">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="QPushButton" name="btn_switch">
       <property name="text">
        <string>switch</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QSpinBox" name="sb_zoom">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3" rowspan="2" colspan="3">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="lab_zoom">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Zoom</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_search">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="te_lyrics"/>
   </item>
   <item row="4" column="0">
    <widget class="QProgressBar" name="pb_progress">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
